src/disco/FixedWidthField/Real/src/parseBase.hpp

```diff
@@ -1,11 +1,12 @@
 template< typename Iterator >
 static uint64_t
 parseBase( Iterator& it, uint16_t& position, bool& success ){
-  uint64_t base = ( success = isdigit( *it ) ) ? (*it - 48) : 0 ;
-  if ( success ){ 
+  uint64_t base = ( success = isdigit( *it ) ) ? (*it - 48) : 0;
```

Comment:
I assume this is std::isdigit ?

Reply:
In parseInteger, you used the character '0' instead of its ascii code (48).

Reply:
> I assume this is std::isdigit ?
 
Functions from cstdlib are in the global namespace, although those symbols are imported into `std` as well.